core/src/main/kotlin/org/evomaster/core/output/service/HttpWsTestCaseWriter.kt

```diff
@@ -210,6 +212,24 @@ abstract class HttpWsTestCaseWriter : ApiTestCaseWriter() {
         lines: Lines,
         baseUrlOfSut: String
     ) {
+
+        val exActions = mutableListOf<HttpExternalServiceAction>()
+
+        // add all used external service actions for the action
+        if (TestWriterUtils.handleExternalService(config)) {
```

Comment:
this `handleExternalService` should be moved directly inside `EMConfig` (see all similar methods there at the bottom).
Also, as it returns a boolean, should be renamed to represent it, eg, `isEnabledExternalServiceMocking`